code/__HELPERS/unsorted.dm

```diff
@@ -1766,4 +1766,13 @@ var/global/list/g_fancy_list_of_types = null
 			. += 4
 			sleep(world.tick_lag*4)
 			//you might be thinking of adding more steps to this, or making it use a loop and a counter var
-			//	not worth it.
\ No newline at end of file
+			//	not worth it.
+
+/proc/make_bit_triplet()

```

Comment:
I mean like..indentation. These are using spaces and not tabs for this entire proc.